src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -123,6 +123,13 @@ std::vector<ActorID> GcsActorScheduler::CancelOnNode(const ClientID &node_id) {
   return actor_ids;
 }
 
+void GcsActorScheduler::CancelLeasingRequest(const ClientID &node_id,
+                                             const ActorID &actor_id) {
+  auto node_it = node_to_actors_when_leasing_.find(node_id);
+  RAY_CHECK(node_it != node_to_actors_when_leasing_.end());
+  node_it->second.erase(actor_id);
```

Comment:
How about add some comments to explain that it is actually effective for RequestWorkerLease callback function?

Reply:
Sounds good!